src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -326,10 +326,20 @@ public function prepare(Request $request): static
     /**
      * Sends HTTP headers.
      *
+     * @param null|positive-int $statusCode The status code to use. Override the statusCode property if set and not null.
+     *
      * @return $this
      */
     public function sendHeaders(): static
     {
+        if (1 > \func_num_args()) {
+            trigger_deprecation('symfony/http-foundation', '6.2', 'Calling "%s()" without any arguments is deprecated, pass null explicitly instead.', __METHOD__);
```

Comment:
should not be necessary: https://github.com/symfony/symfony/pull/48128#issuecomment-1312778921